__tests__/app.test.js

```diff
@@ -76,3 +75,59 @@ describe('/api/articles/:article_id', () => {
             })
     })
 })
+
+describe('api/articles', () => {
+    describe('GET - 200', () => {
+        it('Returns an array of articles', () => {
+            return request(app).get('/api/articles')
+                .expect(200)
+                .then(({ body: { articles }}) => {
+                    articles.forEach(article => {
```

Comment:
test for the length of the array before you use a forEach. If the array is empty then the tests inside your forEach would be skipped but the test for the length would still run